src/bench.c

```diff
@@ -799,8 +800,23 @@ int benchmark_all(void)
 		if ((format->params.flags & FMT_DYNAMIC) ||
 		    strstr(format->params.label, "-opencl") ||
 		    strstr(format->params.label, "-ztex") ||
-		    !strcmp(format->params.label, "crypt"))
+		    !strcmp(format->params.label, "crypt")) {
+#ifdef HAVE_OPENCL
+/*
+ * Allow OpenCL build's "--test" to run on no-OpenCL systems
+ * It is a like a hack, but it is necessary since OpenCL must be started
+ * to get the number of devices. The init[1] itself is performed only once.
+ * [1] Enumeration of platforms and devices; and parse options.
+*/
```

Comment:
I didn't say it explicitly (my bad), but I had suggested some edits to this comment's text (not only indentation). Can you please take my version of the comment from the GitHub review comment where I talked about the indentation? Unless you insist on your version.